Kinvey/Kinvey/User.swift

```diff
@@ -538,7 +551,7 @@ open class User: NSObject, Credential, Mappable {
                 micLoginVC.setValue(forceUIWebView, forKey: "forceUIWebView")
             }
             micLoginVC.client = client
-            micLoginVC.micApiVersion = client.micApiVersion
+            micLoginVC.micApiVersion = client.micApiVersion?.rawValue
```

Comment:
My preference is to use the `rawValue` only when we build the URL. In the rest of the SDK, why don't we pass the enumeration value?